api/src/main/resources/application.properties

```diff
@@ -73,7 +73,7 @@ gov.pay.operational_services=${GOV_PAY_OPERATIONAL_SERVICES:#{ccd_gw,api_gw}}
 idam.s2s-auth.totp_secret=${PAYMENT_S2S_SECRET:AAAAAAAAAAAAAAAA}
 idam.s2s-auth.microservice=${PAYMENT_S2S_MICROSERVICE_NAME:payment_app}
 idam.s2s-auth.url=${AUTH_PROVIDER_SERVICE_CLIENT_BASEURL:#{http://dummy.dummy.net}}
-trusted.s2s.service.names=${TRUSTED_S2S_SERVICE_NAMES:cmc,cmc_claim_store,probate_frontend,divorce_frontend,ccd_gw,bar_api,api_gw,finrem_payment_service,ccpay_bubble,jui_webapp,xui_webapp,fpl_case_service,iac,probate_backend,civil_service,paymentoutcome_web}
+trusted.s2s.service.names=${TRUSTED_S2S_SERVICE_NAMES:refunds_api,cmc,cmc_claim_store,probate_frontend,divorce_frontend,ccd_gw,bar_api,api_gw,finrem_payment_service,ccpay_bubble,jui_webapp,xui_webapp,fpl_case_service,iac,probate_backend,civil_service,paymentoutcome_web}
```

Comment:
make sure we add refunds_api in cnp_flux as well

Reply:
https://github.com/hmcts/cnp-flux-config/pull/13133